docs/HowToInstall.md

```diff
@@ -0,0 +1,74 @@
+# How to install?
+
+## Prerequisite
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### VS Code and its extensions
+
+Let's install vscode.
+- You can download vscode [HERE](https://code.visualstudio.com/download)
+- Official document: [setup](https://code.visualstudio.com/docs/setup/setup-overview)
```

Comment:
Q) Is there a reason of `HERE` and `setup`? How about to unify `HERE` and `setup` to one of lowercase or uppercase?

Reply:
```suggestion
- You can download vscode [here](https://code.visualstudio.com/download)
- Refer [official document](https://code.visualstudio.com/docs/setup/setup-overview) for installing vscode.
```